completions/bash/podman

```diff
@@ -3354,6 +3365,7 @@ _podman_pod() {
      stop
      top
      unpause
+	 inspect
```

Comment:
space/tab issue?

Reply:
:fearful: OMG - could that be?!
Thanks @TomSweeneyRedHat ! :bow: